src/e2e-test/features/bigquery/sink/GCSToBigQuery.feature

```diff
@@ -7,8 +7,8 @@ Feature: BigQuery sink - Verification of GCS to BigQuery successful data transfe
     When Source is GCS
     When Sink is BigQuery
     Then Open GCS source properties
-    Then Override Service account details if set in environment variables
     Then Enter the GCS source mandatory properties
+    Then Override Service account details if set in environment variables
```

Comment:
Why reordering of this step is needed??

Reply:
explained in the description. The first step will trigger the `get schema` and it will fail without the service account json.

Reply:
ahh makes sense, from the description.